packages/kbn-router-to-openapispec/src/process_router.ts

```diff
@@ -64,11 +64,13 @@ export const processRouter = (
         parameters.push(...pathObjects, ...queryObjects);
       }
 
-      let description = '';
+      let description = `${route.options.description ?? ''}`;
       if (route.security) {
         const authzDescription = extractAuthzDescription(route.security);
 
-        description = `${route.options.description ?? ''}${authzDescription ?? ''}`;
+        description += `${route.options.description && authzDescription ? `<br/><br/>` : ''}${
```

Comment:
Searching for newline implementation for OAS yaml spec, i came across both `<br/> and \n`. However, I used the former because the new line wouldn't get rendered correctly with `\n` - for example:
<img width="801" alt="image" src="https://github.com/user-attachments/assets/cf8c71dc-579a-43a2-ab5f-69db081b13d8">